lib/promise.js

```diff
@@ -2,5 +2,11 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+'use strict'
+
 // for easy promise lib switching
 module.exports = require('bluebird')
+
+process.on('unhandledRejection', (reason, promise) => {
```

Comment:
The route in `sign.js` was through exceptions about using `reply` more than once, and we never dealt with the error or even really knew about it other than an odd hapi debug output. This change made it so unhandled rejections were treated as uncaught exceptions. While it might mean more crashes, it also means no errors that go unnoticed.